source/time.tex

```diff
@@ -8039,713 +8077,340 @@
-$hours$:$minutes$:$seconds$.
-The \tcode{Duration} template parameter dictates
-the precision to which the time is broken down.
-\begin{note}
-This can vary from a coarse precision of hours
-to a very fine precision of nanoseconds.
-\end{note}
-A \tcode{time_of_day} object also tracks
-whether it should be output
-as a 12-hour time format or a 24-hour time format.
+  template <class Duration>
+  class hh_mm_ss
+  {
+    bool            is_neg;     // \expos
+    chrono::hours   h;          // \expos
+    chrono::minutes m;          // \expos
+    chrono::seconds s;          // \expos
+    precision       ss;         // \expos
 
-\pnum
-The primary \tcode{time_of_day} template is not defined.
-Four specializations are provided to handle four different
-levels of precision.
-
-\pnum
-Each specialization of \tcode{time_of_day} is a trivially copyable
-and standard-layout class type.
-
-\rSec2[time.tod.hours]{Hours precision}
-\indexlibrary{\idxcode{time_of_day<hours>}}
-
-\begin{codeblock}
-namespace std::chrono {
-  template<>
-  class time_of_day<hours> {
   public:
-    using precision = chrono::hours;
+    static unsigned constexpr fractional_width = @\seebelow@;
+    using precision                            = @\seebelow@;
 
-    time_of_day() = default;
-    constexpr explicit time_of_day(chrono::hours since_midnight) noexcept;
+    constexpr hh_mm_ss() noexcept : hh_mm_ss{Duration::zero()} {}
+    constexpr explicit hh_mm_ss(Duration d);
 
+    constexpr bool is_negative() const noexcept;
     constexpr chrono::hours hours() const noexcept;
+    constexpr chrono::minutes minutes() const noexcept;
+    constexpr chrono::seconds seconds() const noexcept;
+    constexpr precision subseconds() const noexcept;
 
-    constexpr explicit operator  precision()   const noexcept;
-    constexpr          precision to_duration() const noexcept;
-
-    constexpr void make24() noexcept;
-    constexpr void make12() noexcept;
+    constexpr explicit operator precision() const noexcept;
+    constexpr precision to_duration() const noexcept;
   };
+
+  template <class charT, class traits, class Duration>
+    basic_ostream<charT, traits>&
+      operator<<(basic_ostream<charT, traits>& os, hh_mm_ss<Duration> const& hms);
 }
 \end{codeblock}
 
 \pnum
-\begin{note}
-This specialization handles hours since midnight.
-\end{note}
-
-\indexlibrary{\idxcode{time_of_day<hours>}!constructor}%
-\begin{itemdecl}
-constexpr explicit time_of_day(chrono::hours since_midnight) noexcept;
-\end{itemdecl}
-
-\begin{itemdescr}
-\pnum
-\effects
-Constructs an object of type \tcode{time_of_day} in 24-hour format
-corresponding to \tcode{since_midnight} hours after 00:00:00.
-
-\pnum
-\ensures
-\tcode{hours()} returns the integral number of hours \tcode{since_midnight} is after 00:00:00.
-\end{itemdescr}
-
-\indexlibrarymember{hours}{time_of_day<hours>}%
-\begin{itemdecl}
-constexpr chrono::hours hours() const noexcept;
-\end{itemdecl}
+The \tcode{hh_mm_ss} class template
```

Comment:
All this sounds like description and non-normative wording, and thus should be moved before the class synopsis.

Reply:
Having the description after the class synopsis is locally consistent.

I won't change this.